torch/_inductor/compile_fx.py

```diff
@@ -2380,10 +2380,9 @@ def partition_fn(
             with dynamo_utils.dynamo_timed(
                 "min_cut_rematerialization_partition", log_pt2_compile_event=True
             ):
-                return min_cut_rematerialization_partition(
+                return default_partition(
```

Comment:
This is clearly not OK, not sure what the right way to "configure" inductor to do this. Likely a more taggy approach with the min cut is better.